pkg/validation/app.go

```diff
@@ -174,6 +180,20 @@ func (v *Validator) validateName(ctx context.Context, cr v1alpha1.App) error {
 	return nil
 }
 
+// We make sure users cannot create in-cluster Apps outside their organization
+// or WC namespaces. Otherwise `.spec.namespace` could be exploited to override permissions.
+func (v *Validator) validateTargetNamespace(ctx context.Context, cr v1alpha1.App) error {
+	if key.InCluster(cr) {
```

Comment:
These three if statements could be combined into one - but not a big thing.

Reply:
Does it look better now? I had a nested statements cause they looked more readable then a single, but long one, but after combining them it doesn't look less elegant 😄

Reply:
Looks good to me!